src/sentry/constants.py

```diff
@@ -233,6 +235,94 @@ def get_all_languages():
 NATIVE_UNKNOWN_STRING = '<unknown>'
 
 
+# to go from an integration id (in _platforms.json) to the platform
+# data, such as documentation url or humanized name.
+# example: java-logback -> {"type": "framework",
+#                           "link": "https://docs.getsentry.com/hosted/clients/java/modules/logback/",
+#                           "id": "java-logback",
+#                           "name": "Logback"}
+INTEGRATION_ID_TO_PLATFORM_DATA = {}
+
+
+def _load_platform_data():
+    INTEGRATION_ID_TO_PLATFORM_DATA.clear()
+    data = load_doc('_platforms')
+
+    if not data:
+        return
+
+    for platform in data['platforms']:
+        integrations = platform.pop('integrations')
+        if integrations:
+            for integration in integrations:
+                integration_id = integration.pop('id')
+                INTEGRATION_ID_TO_PLATFORM_DATA[integration_id] = integration
+
+_load_platform_data()
+
+
+# special cases where the marketing slug differs from the integration id
+# (in _platforms.json). missing values (for example: "java") should assume
+# the marketing slug is the same as the integration id.
+MARKETING_SLUG_TO_INTEGRATION_ID = {
+    "kotlin": "java",
+    "scala": "java",
+    "android": "java-android",
+
+    # TODO: add more special cases...
```

Comment:
Ya. Unless we get all SDKs to send some kind of unique id (and I can think of events that would have more than 1 id, so this gets weird) I think it has to be maintained somewhere. :(